src/context/fetch.ts

```diff
@@ -26,8 +26,13 @@ const createFetchRequestParameters = (input: MockedRequest): RequestInit => {
     return requestParameters
   }
 
-  requestParameters.body =
-    typeof body === 'object' ? JSON.stringify(body) : body
+  if (typeof body === 'object') {
+    requestParameters.body = JSON.stringify(body)
+  } else if (typeof body === 'number') {
+    requestParameters.body = body.toString()
```

Comment:
Converting number to string and stringifiying it via `JSON` yields the same result:

```js
number.toString() === JSON.stringify(number) // true
```

I suggest we revert this particular change. What do you think about this?

Reply:
Can you please elaborate on what usage scenario are you covering with this change?

A `MockedRequest` instance should be well-formed when it's passed to `ctx.fetch`. It's a request representation, which, as we've discussed, cannot indeed have a number body (it'll always be a string). Perhaps I'm missing something. Having a test for this would be great.

Reply:
I've added the integration tests for this change. It does behave correctly:

1. You can respond with a number in a mocked response: `ctx.json(123)`. The number is then properly extracted when you read the response as JSON.
2. You can send a number in a JSON request body (using a regular POST request). The number is then properly presented in `req.body` of your request handler.